src/components/category/Categories.tsx

```diff
@@ -0,0 +1,56 @@
+import styles from '../category/Categories.module.css';
+import { useSelector, useDispatch } from 'react-redux';
+import { ChangeEvent, useState } from 'react';
+import { changeCategoryType } from '../../redux/slices/filterSlice';
+import { categoryType, selectDucksFiltered } from '../../redux/selectors';
+import { RootState } from '../../redux/store';
+
+export function Categories() {
+  const dispatch = useDispatch();
+  const ducksData = useSelector(selectDucksFiltered);
+  const prevDD = useSelector((state: RootState) => state.ducks.ducks);
+
+  const categoryR = useSelector(categoryType);
+
+  const [check, setCheck] = useState(categoryR);
```

Comment:
[eslint] React Hook useEffect has a missing dependency: 'navigate'. Either include it or remove the dependency array